style.css

```diff
@@ -0,0 +1,568 @@
+@import url("https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap");
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+  font-family: "Poppins", sans-serif;
+  background: #f4f5f7;
+}
```

Comment:
- [ ] Since all the styling after this line is for the desktop version. currently, you have none of the styling that was for the mobile version. The requirement is for you to implement the desktop version styling while retaining all the styling for the mobile version. I would suggest that you find the styling that you used for the mobile and bring it back so that both the desktop and mobile versions work together. 
- [ ] Please ensure that the mobile version looks the way it previously did matching the Figma in the same way as before and doesn't affect the desktop version.
- [ ] Ensure that the styling for the desktop version also doesn't interfere with the styling of the desktop version.

Your Design | Reference Design
-|-
![image](https://user-images.githubusercontent.com/18730106/155596804-01a1d737-12bb-4a53-a742-943438ac1ff5.png)|![image](https://user-images.githubusercontent.com/18730106/155596856-1dbcc748-7672-4fe6-9a12-461460c1450a.png)|